### ISSUE TITLE
Create new Source Note heading (Inventory)

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#509]
Please could you create a new Source Note category of 'Inventory' -- or if you want to be specific, 'source of inventory'. However 'Inventory' would be better in case we want to paste a flat inventory in to there until we can make a proper one. This is because we have had some submitted by individuals, and those can be attributed in the commentary tab, but this is not ideal as it doesn't give them much credit for what is really a very large piece of work. However the other reason is that if we find an inventory ourselves from e.g. an article, we should record where we got it from somewhere immediately visible (and where people would expect to see that information), so a Source Note would make sense. Of course ideally, that note should appear at the top of the inventory, but that is probably a bridge too far. Just having another heading for source notes is the best and I hope quickest solution.

I actually thought it was already there, but perhaps it was removed? We did have some flat text inventories, but they may have been lost.

**Comment by ahankinson** [comment]
I think this is what the "Index" note type is for? Would that work?

**Comment by juliacmcf** [comment]
Nope: that’s for a contemporary index at the front of the book, written by the original scribe, and allows us to transcribe the original index made by the scribe, or just make comments about it’s e.g. incompleteness, or the fact that it lists pieces that no longer survive.
